scripts/install_moodle.sh

```diff
@@ -422,77 +460,77 @@ echo_title "Run Moodle Installer."
-    --dbpass="${parameters[moodleDbPassword]}" \
-    --fullname="Moodle" \
-    --shortname="Moodle" \
-    --summary="Welcome - Bienvenue" \
-    --adminuser=${parameters[moodleAdminUsername]} \
-    --adminpass="${parameters[moodleAdminPassword]}" \
-    --adminemail=${parameters[moodleAdminEmail]} \
-    --upgradekey=${parameters[moodleUpgradeKey]} \
-    ${skipDatabaseOption} \
-    --agree-license
+sudo -u ${APACHE2_USER} /usr/bin/php ${MOODLE_DOCUMENT_ROOT_DIR_PATH}/admin/cli/install.php \
+  --non-interactive \
+  --lang=en \
+  --chmod=2777 \
+  --wwwroot=https://${parameters[moodleFqdn]}/ \
+  --dataroot=${parameters[moodleDataMountPointPath]}/ \
+  --dbtype=pgsql \
+  --dbhost=${parameters[dbServerFqdn]} \
+  --dbname=${parameters[moodleDbName]} \
+  --prefix=$table_prefix \
+  --dbport=5432 \
+  --dbuser=${parameters[moodleDbUsername]}@${parameters[dbServerName]} \
+  --dbpass="${parameters[moodleDbPassword]}" \
+  --fullname="Moodle" \
+  --shortname="Moodle" \
+  --summary="Welcome - Bienvenue" \
+  --adminuser=${parameters[moodleAdminUsername]} \
+  --adminpass="${parameters[moodleAdminPassword]}" \
+  --adminemail=${parameters[moodleAdminEmail]} \
+  --upgradekey=${parameters[moodleUpgradeKey]} \
+  ${skip_database_option} \
+  --agree-license
 echo_info "Done."
 
 ###############################################################################
 echo_title "Moodle Post installation process."
 ###############################################################################
 # No need to test for existing values since the file is always new.
-echo_action "Adding SSL Proxy setting to ${moodleDocumentRootDirPath}/config.php file..."
-sed -i '/^\$CFG->wwwroot.*/a \$CFG->sslproxy\t= true;' ${moodleDocumentRootDirPath}/config.php
+echo_action "Adding SSL Proxy setting to ${MOODLE_DOCUMENT_ROOT_DIR_PATH}/config.php file..."
+sed -i '/^\$CFG->wwwroot.*/a \$CFG->sslproxy\t= true;' ${MOODLE_DOCUMENT_ROOT_DIR_PATH}/config.php
 echo_info "Done."
 
-echo_action "Adding Local Cache Directory setting to ${moodleDocumentRootDirPath}/config.php file..."
-sed -i "/^\$CFG->dataroot.*/a \$CFG->localcachedir\t= '${moodleLocalCacheRootDirPath}';" ${moodleDocumentRootDirPath}/config.php
+echo_action "Adding Local Cache Directory setting to ${MOODLE_DOCUMENT_ROOT_DIR_PATH}/config.php file..."
+sed -i "/^\$CFG->dataroot.*/a \$CFG->localcachedir\t= '${MOODLE_LOCAL_CACHE_DIR_PATH}';" ${MOODLE_DOCUMENT_ROOT_DIR_PATH}/config.php
 echo_info "Done."
 
-echo_action "Adding default timezone setting to ${moodleDocumentRootDirPath}/config.php file..."
-sed -i "/^\$CFG->upgradekey.*/a date_default_timezone_set('America/Toronto');" ${moodleDocumentRootDirPath}/config.php
+echo_action "Adding default timezone setting to ${MOODLE_DOCUMENT_ROOT_DIR_PATH}/config.php file..."
+sed -i "/^\$CFG->upgradekey.*/a date_default_timezone_set('America/Toronto');" ${MOODLE_DOCUMENT_ROOT_DIR_PATH}/config.php
 echo_info "Done."
 
```

Comment:
It might be good to add the debug options to the config file for the DEV environment at this step

Reply:
@error_reporting(E_ALL | E_STRICT);   // NOT FOR PRODUCTION SERVERS!
@ini_set('display_errors', '1');         // NOT FOR PRODUCTION SERVERS!
$CFG->debug = (E_ALL | E_STRICT);   // === DEBUG_DEVELOPER - NOT FOR PRODUCTION SERVERS!
$CFG->debugdisplay = 1;              // NOT FOR PRODUCTION SERVERS!

Reply:
Suggestion added to issue list.